src/xcode/ENA/ENA/Source/Developer Menu/Features/DMLastRiskCalculationViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+#if !RELEASE
+
+import UIKit
+
+class DMLastRiskCalculationViewController: UIViewController {
+	init(lastRisk: String?) {
+		self.lastRisk = lastRisk
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func loadView() {
+		view = textView
+		view.backgroundColor = .white
+	}
+
+	private let lastRisk: String?
+	private let textView = UITextView()
+
+	override func viewWillAppear(_ animated: Bool) {
+		textView.attributedText = NSAttributedString(string: lastRisk ?? "")
```

Comment:
I have now added:

```
exportItem.isEnabled = lastRisk != nil
```

So I think having this property optional is useful because the `isEnabled`-check is a bit clearer when there is no default value and we simply use `nil` for that. Using `lastRisk != ""` as a check seems kind of like the same as using a magic value...